sphinx/util/inspect.py

```diff
@@ -149,7 +144,7 @@ def getslots(obj: Any) -> Optional[Dict]:
 
 def isNewType(obj: Any) -> bool:
     """Check the if object is a kind of NewType."""
-    if sys.version_info >= (3, 10):
+    if sys.version_info[:2] >= (3, 10):
```

Comment:
I think all these `[:2]` additions are unnecessary for `>`, `>=` and `<`?

Or is it for consistency so we don't need to worry about `<=`?

Reply:
I always forget which operator is the special case so personally adding `[:2]` gives me more reassurance, you're likely right that is isn't strictly necessary.

A